apps/sage/account/NanoList.tsx

```diff
@@ -38,41 +24,71 @@ export default function NanoList() {
         a.href = window.URL.createObjectURL(data)
         a.download = 'registration.zip'
         a.click()
-      }).catch(err => console.log(err.message))
+      }).catch(err => {
+        if (err.message) {
+          setRegKey(false)
+          setErrMsg(err.message)
+          console.log(err.message)
```

Comment:
As discussed, this is logging is unnecessary.  If a developer wanted to see the message, they can use either the dev tools > network tab, "log all http requests", or similar.